### ISSUE TITLE
Forced look after copyover should be screenreader toggled

### ISSUE THREAD

**Issue by luciensadi** [luciensadi/AwakeMUD#32]
Really, anything that forces large volumes of text on the client should be toggled.

**Comment by jalles76** [comment]
Quickwalk macros come to mind. It should even be possible to temporarily toggle the output of room descriptions.

**Comment by luciensadi** [comment]
Forced look is screenreader toggled. Issues should be filed separately for other things that need to be toggled as well.
